docs/source/author/understanding_configuration_files.rst

```diff
@@ -0,0 +1,68 @@
+Understanding configuration files
+##############################################################################
+
+There is a lot of content within the CS Unplugged project.
+We split this content across many files and configuration files are the things
+that bring everything together.
+Understanding how to read and write these configuration files is important
+for working on this project.
+
+We currently use `YAML`_ for storing configuration data of content.
+It has improved human readability over JSON and XML, especially for authors
+who have no or little experience with configuration files.
```

Comment:
I just don't think this needs to be in author docs. Author docs should be a safe place where there is very minimal dev speak. These kinds of justifications should go in developer docs.